assets/stylesheets/test-details.scss

```diff
@@ -414,4 +414,14 @@ ul.modcategory {
     .fa-download:hover {
         opacity: inherit;
     }
+}
+
+.test-result-overview-link {
+    top: -0.1em;
```

Comment:
Can you move that one down so the relative coordinates are specified consecutively?

Reply:
done